app/scripts/modules/core/delivery/executionGroup/ExecutionGroup.tsx

```diff
@@ -144,18 +146,18 @@ export class ExecutionGroup extends React.Component<IProps, IState> {
     }
   }
 
-  private handleHeadingClicked = () => {
+  private handleHeadingClicked(): void {
```

Comment:
The only upside of adding `: void` return types is it makes it more clear when methods that return something else are missing a return type, so I'm having mixed feelings about it...